srv/SpawnEntity.srv

```diff
@@ -0,0 +1,19 @@
+# Spawn an entity (a robot, other object) by name or URI
+
+string name                         # Suggest a name to give to the spawned entity. If left empty, it will be generated.
+                                    # Due to requirement for name uniqueness and the namespace, check the status_message
+                                    # to get an actual unique name to use for further queries.
```

Comment:
I'm not sure how many people will look at this documentation and know that they're supposed to get the unique name from the returned message of this service. I'd imagine most users will call SpawnEntity and assume that if there was no error, that the entity was spawned with the name they provided. If there was a name collision and the simulator decided to generate a unique name, their subsequent interaction will be with the wrong entity. I think this is why by default, Gazebo returns an error by default if the name is not unique and has an `allow_renaming` flag to opt into name generation by the simulator.